cpp/ql/test/query-tests/Likely Bugs/Memory Management/ReturnStackAllocatedMemory/test.cpp

```diff
@@ -248,4 +248,9 @@ char* test_strdupa(const char* s) {
 void* test_strndupa(const char* s, size_t size) {
 	char* s2 = strndupa(s, size);
 	return s2; // BAD
-}
\ No newline at end of file
+}
+
+UNKNOWN_TYPE test_error_type() {
+	UNKNOWN_TYPE x;
+	return x; // GOOD: Don't report error types
+}
```

Comment:
I don't recall what we did in other cases, but it might be better to have all test cases with parse errors in separate files, because that will mean we catch parse errors that are introduced by accident in "regular" tests.

Reply:
That's not a bad idea to start doing that.